test/e2e/handler/metrics.go

```diff
@@ -0,0 +1,34 @@
+package handler
+
+import (
+	"strings"
+
+	"github.com/nmstate/kubernetes-nmstate/test/cmd"
+	"github.com/nmstate/kubernetes-nmstate/test/runner"
+)
+
+func getMetrics(token string) string {
+	bearer := "Authorization: Bearer " + token
+	return runner.RunAtFirstHandlerPod("curl", "-s", "-k", "--header", bearer, "https://127.0.0.1:8443/metrics")
```

Comment:
you can use `metrics.DefaultBindAddress` here too instead of the port hardcoded value

Reply:
I am not sure the address here is not `:8080`

Reply:
you also have `8443` configured in the service you created